#include "gsteam_client_player.h"
#include "src/game_logic/game_registry.h"
#include "src/network/message_sys.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void ClientPlayerTeamServiceImpl::TeamInfoNotify(EntityPtr& entity,
    const ::TeamInfoS2CRequest* request,
    ::TeamInfoS2CResponse* response)
{
///<<< BEGIN WRITING YOUR CODE TeamInfoNotify
///<<< END WRITING YOUR CODE TeamInfoNotify
}

///<<<rpc end
